src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ListGroup, Card } from 'react-bootstrap';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo: React.FC<PreparedPost> = ({
+  id,
+  title,
+  user,
+  body,
+  comments,
+}) => (
+  <>
+    <ListGroup.Item as="li" key={id}>
+      <Card>
+        <Card.Body>
+          <Card.Title>{title}</Card.Title>
+          <Card.Text>{body}</Card.Text>
+          <UserInfo user={user} />
```

Comment:
check if the user will be not null, then render it, because now you will have just empty HTML elements